### PR TITLE
Initial real commit with skeletal Post index view and creation form

### PR DESCRIPTION
**Pull request by joeyAghion**
This repo's initial commit (https://github.com/joeyAghion/hokusai_demo/commit/9b45925cd142a1c94be4585c3ef8ac01d7cc20ab) consists of only what's generated by:

    rails new hokusai_demo --database=postgresql

This PR adds some minimal stuff to actually run the project locally, and basic list and create views for a skeletal `Post` model. Also:

* removed `spring` to avoid cached dependency headaches
* tweaked the `database.yml` to work with typical local postgres
* added a basic integration test exercising the new actions

Basically my operations were:

    rake db:setup
    rake db:migrate
    rails g model Post # add columns to generated migration...
    spring stop # remove spring-related gems from Gemfile and re-bundle...
    rails g controller posts # add controller actions, update routes.rb, create views...

CC: @cavvia

### CODE REVIEW

**Comment by cavvia** [comment]
ta - i was expecting to see some hokusai generated yaml files in here but this is good thanks.
